pallets/delegation/src/lib.rs

```diff
@@ -87,7 +87,7 @@ use frame_support::{
 use kilt_support::traits::StorageDepositCollector;
 use parity_scale_codec::Encode;
 use sp_runtime::{traits::Hash, DispatchError};
-use sp_std::{marker::PhantomData, vec::Vec};
+use sp_std::{marker::PhantomData, vec, vec::Vec};
```

Comment:
This dependendency is only used in `try-runtime`, right? In this case it should not be pulled in regardless, and also it could either be used directly with `sp_std::vec`, or imported inside the function that needs it.